task/sql-tasks.js

```diff
@@ -177,7 +275,17 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT 
+        (SELECT COUNT(ProductID))as TotalOfCurrentProducts, 
+        (
+            SELECT COUNT(Discontinued)
+            FROM Products
+            WHERE Discontinued > 0
+        ) as TotalOfDiscontinuedProducts
+    FROM Products

```

Comment:
from-clause is optional in this case it's better to just use 2 subqueries

Reply:
sorry i didn't understand, i have to use 2 subqueries. maybe it's wrong code formatting.

Reply:
```
select
   subquery1 as 'TTT'
   subquery2  as 'TTTTT'
<!--- the end -->

```